<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TDialogData</class>
 <widget class="QDialog" name="TDialogData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1" rowspan="2">
         <widget class="QSpinBox" name="spinEmpNo">
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
        <item row="6" column="1" rowspan="2">
         <widget class="QDateEdit" name="editBirth">
          <property name="dateTime">
           <datetime>
            <hour>16</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2017</year>
            <month>2</month>
            <day>19</day>
           </datetime>
          </property>
         </widget>
        </item>
        <item row="4" column="1" rowspan="2">
         <widget class="QComboBox" name="comboSex">
          <item>
           <property name="text">
            <string>男</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>女</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="10" column="0" rowspan="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>部门</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1" rowspan="3">
         <widget class="QComboBox" name="comboProvince">
          <item>
           <property name="text">
            <string>北京</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>上海</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>广西</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>深圳</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>杭州</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>广州</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1" rowspan="2">
         <widget class="QLineEdit" name="editName"/>
        </item>
        <item row="11" column="1">
         <widget class="QComboBox" name="comboDep">
          <item>
           <property name="text">
            <string>销售部</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>技术部</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>管理部</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QDoubleSpinBox" name="spinSalary">
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
          <property name="value">
           <double>1000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>工资</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>出生省份</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>出生日期</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>性别</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>性名</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>工号</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>照片</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="LabPhoto">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>dbLabPhoto</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>导入照片</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/images/00.JPG</normaloff>:/icons/images/00.JPG</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>清除照片</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/images/103.bmp</normaloff>:/icons/images/103.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/images/704.bmp</normaloff>:/icons/images/704.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/images/706.bmp</normaloff>:/icons/images/706.bmp</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>备注</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="editMemo"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>TDialogData</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>TDialogData</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
